sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -2526,10 +2526,10 @@ object DatePart {
        224
       > SELECT _FUNC_('SECONDS', timestamp'2019-10-01 00:00:01.000001');
        1.000001
-      > SELECT _FUNC_('days', interval 1 year 10 months 5 days);
+      > SELECT _FUNC_('days', interval 5 days 3 hours 7 minutes);
        5
-      > SELECT _FUNC_('seconds', interval 5 hours 30 seconds 1 milliseconds 1 microseconds);
```

Comment:
this example doesn't need to be changed.

Reply:
`milliseconds` and `microseconds` are not valid ANSI interval fields are they?

Reply:
Ah, I guess you mean `milliseconds` and `microseconds` aren't ANSI supported fields but it's acceptable expansion to mix valid ANSI day-time fields and non-ANSI ones (`WEEK`, `MILLISECOND` and `MICROSECOND`) right?

Reply:
yup